internal/meta/sinkMeta.go

```diff
@@ -118,9 +118,8 @@ func newField(fis []*fileField) (uis []field, err error) {
 			Hint:              newLanguage(fi.Hint),
 			Label:             newLanguage(fi.Label),
 		}
-		uis = append(uis, ui)
 		switch t := fi.Default.(type) {
-		case []map[string]interface{}:
+		case []interface{}:
```

Comment:
Is it possible not this type

Reply:
not possible, the fi is processed result of json